rebar.config

```diff
@@ -1,21 +1,21 @@
 {erl_opts, [debug_info]}.
 {deps, [
-	{capwap, ".*", {git, "git://github.com/travelping/capwap", {tag, "1.9.1"}}},
-	{gen_listener_tcp, ".*", {git, "git://github.com/travelping/gen_listener_tcp", "master"}},
-	{netdata, ".*", {git, "git://github.com/RoadRunnr/erl_netdata", "master"}}
+    {capwap,           {git, "git://github.com/travelping/capwap",
+                           {branch, "master"}}},
+    {gen_listener_tcp, {git, "git://github.com/travelping/gen_listener_tcp",
+                           {branch, "master"}}},
+    {netdata,          {git, "git://github.com/RoadRunnr/erl_netdata",
+                           {branch, "master"}}}
 ]}.
 
-{relx, [{release, { 'ergw-capwap-node', "1.9.1" },
-         [capwap, netdata, sasl, runtime_tools]},
-
-        {sys_config, "./config/sys.config"},
-        {vm_args, "./config/vm.args"},
-
-        {dev_mode, true},
-        {include_erts, false},
-
-        {extended_start_script, true}]
-}.
+{relx, [
```

Comment:
I don't like the whitespace only changes

Reply:
adjusted

Reply:
the whitespace in that file is not really in one style or the other...